pinot-broker/src/test/java/com/linkedin/pinot/broker/broker/HelixBrokerStarterTest.java

```diff
@@ -123,10 +125,14 @@ public void testResourceAndTagAssignment() throws Exception {
         _helixAdmin.getResourceExternalView(HELIX_CLUSTER_NAME, CommonConstants.Helix.BROKER_RESOURCE_INSTANCE);
     Assert.assertEquals(externalView.getStateMap(COMPANY_TABLE_NAME).size(), 6);
     Thread.sleep(2000);
-    HelixExternalViewBasedRouting helixExternalViewBasedRouting =
-        _helixBrokerStarter.getHelixExternalViewBasedRouting();
-    Assert
-        .assertEquals(Arrays.toString(helixExternalViewBasedRouting.getDataTableSet().toArray()), "[company_OFFLINE]");
+    HelixExternalViewBasedRouting helixExternalViewBasedRouting = _helixBrokerStarter.getHelixExternalViewBasedRouting();
+    Field brokerRoutingTableField;
+    brokerRoutingTableField = HelixExternalViewBasedRouting.class.getDeclaredField("_brokerRoutingTable");
+    brokerRoutingTableField.setAccessible(true);
```

Comment:
Reflection is used only for testing, so that we can get more code coverage. Adding a getter just for this purpose seems useless. If we need to re-factor the class, then we need to change the unit tests as well -- whether we do it via getter or reflection